### PR TITLE
config: skip lengthy values

### PR DESCRIPTION
**Pull request by greut**
Passing three more tests regarding maximum length for section name, properties and values.

> The maximum length of a section name is 4096 characters. All sections exceeding this limit are ignored.
> [...]
> The maximum length of a property name is 50 characters and the maximum length of a property value is 255 characters. Any property beyond these limits would be ignored.
>
> https://editorconfig-specification.readthedocs.io/en/latest/

### CODE REVIEW

File: editorconfig.go [hunk 1]
```diff
@@ -42,6 +42,13 @@ const (
 	CharsetUTF8BOM = "utf-8 bom"
 )
 
+// Limits for section name, properties, and values.
+const (
+	MaxPropertyLength = 50
+	MaxSectionLength  = 4096
+	MaxValueLength    = 255
```
Comment (andreynering) [inline editorconfig.go]: Just a question we should make here (no strong opinion on this, though): should these constants really be exported? Are they useful for the user?

Comment (greut) [reply editorconfig.go]: I don't see any problems exporting those *magic numbers*. I followed the way it's been done above.

**Comment by greut** [comment]
```
97% tests passed, 6 tests failed out of 193

Total Test time (real) =   0.49 sec

The following tests FAILED:
        123 - indent_size_default_pre_0_9_0 (Failed)
        148 - comments_after_section (Failed)
        155 - octothorpe_comments_after_section (Failed)
        158 - octothorpe_in_property (Failed)
        159 - escaped_octothorpe_in_property (Failed)
        173 - root_file_mixed_case (Failed)
```
